graylog2-web-interface/src/@types/graylog-web-plugin/index.d.ts

```diff
@@ -193,6 +193,7 @@ declare module 'graylog-web-plugin/plugin' {
     // Global context providers allow to fetch and process data once
     // and provide the result for all components in your plugin.
     globalContextProviders?: Array<React.ComponentType<React.PropsWithChildrean<{}>>>,
+    pageContextProviders?: Array<React.ComponentType<React.PropsWithChildrean<{}>>>,
```

Comment:
Can you add a comment to describe the relation between `globalContextProviders` and  `pageContextProviders` so it is easier to understand hen to use what?